src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -251,7 +251,7 @@ static bool tryCastStringToStruct(const char* input, uint64_t len, ValueVector*
         skipWhitespace(++input, end);
 
         copyStringToVector(StructVector::getFieldVector(vector, fieldIdx).get(), rowToAdd,
-            std::string_view{valStart, valEnd}, csvReaderConfig);
+            std::string_view{valStart, (size_t)(valEnd-valStart)}, csvReaderConfig);
```

Comment:
Should we use size_t here? Or uint64? I think we never use size_t in the code base